src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Webservice/index.html.twig

```diff
@@ -58,15 +58,54 @@
     </div>
   </div>
 
+  {% block webservice_api_status %}
+    {% set devdocUrl = 'https://devdocs.prestashop.com/1.7/development/webservice/' %}
+
+    {% if webserviceStatus['enabled'] == true %}
+      <div class="card">
+        <h3 class="card-header">
+          <i class="material-icons">info_outline</i> {{ 'Webservice status'|trans }}
+        </h3>
+        <div class="card-block">
+          <div class="card-text">
+            <p>
+              {{ 'Webservice is enabled. Main entry point is' |trans }}
+              <a href="{{ webserviceStatus['endpoint'] }}">
+                {{ webserviceStatus['endpoint'] }}
+              </a>
+            </p>
+            {% if webserviceStatus['isFunctional'] == true %}
+              <p>
+                {{ "Developer documentation can be found [1]here[/1]."|trans({
+                  '[1]': '<a href="'~devdocUrl~'">', '[/1]': "</a>"
+                }, 'Admin.Advparameters.Feature')|raw }}
```

Comment:
```suggestion
                {{ "You can find the developer documentation [1]here[/1]."|trans({
                  '[1]': '<a href="'~devdocUrl~'">', '[/1]': "</a>"
                }, 'Admin.Advparameters.Feature')|raw }}
```

Reply:
Is it necessary to have another line here with the link to the developer documentation since it is already mentioned in the small block? 🤔

Reply:
Personally, I don't think this line is necessary as the info is already mentioned above like Louise said. I think we can delete it, @matks wdyt? :)

Reply:
You mean the "help" block that appears on the right of Back-Office when clicking on it ?

Reply:
![image](https://user-images.githubusercontent.com/64894223/87306311-204c9c00-c518-11ea-8bfd-74a71857acd1.png)

@matks For the text that appears here!

Reply:
@Junebyun The 2 texts will not appear together. One is displayed when API Endpoint works, the other is displayed when API Endpoint is broken. However I think both texts should contain the documentation link I think.

See here how the 2 texts are displayed https://github.com/PrestaShop/PrestaShop/pull/19335#issuecomment-648015337

Reply:
Hi @matks, since @Junebyun is quite busy with the Gitbook migration, I've taken over the copywriting part. In this case, my suggestion is to use the same wording:

                {{ "Read the [1]developer documentation[/1] to learn more about it."|trans({
                  '[1]': '<a href="'~devdocUrl~'">', '[/1]': "</a>"
                }, 'Admin.Advparameters.Feature')|raw }}